docs/toc.json

```diff
@@ -16,6 +16,15 @@
 			{ "docs/designers-developers/developers/filters/parser-filters.md": [] },
 			{ "docs/designers-developers/developers/filters/autocomplete-filters.md": [] }
 		] },
+		{"docs/designers-developers/developers/slotfills/README.md": [
```

Comment:
It doesn't look like GitHub supports multi-line suggestions, so: the order of this sections affects the _previous_ / _next_ links in the handbook, so I rather have the same order (alphabetical) that we used ([here](https://github.com/WordPress/gutenberg/pull/15738/files#diff-ca7791dcf6533a4ca89570d1e918e1d1R97)).

Reply:
Note that this will require a `npm run docs:build` so the `manifest-devhub.json` is updated.

Reply:
Updated